util/imagetools/loader.go

```diff
@@ -126,13 +127,8 @@ func (l *loader) Load(ctx context.Context, ref string) (*result, error) {
 		}
 
 		var a asset
-		annotations := make(map[string]string, len(mfst.manifest.Annotations)+len(mfst.desc.Annotations))
-		for k, v := range mfst.desc.Annotations {
-			annotations[k] = v
-		}
-		for k, v := range mfst.manifest.Annotations {
-			annotations[k] = v
-		}
+		annotations := maps.Clone(mfst.desc.Annotations)
```

Comment:
Same issue with `nil`

Reply:
Interesting that suggested fix was `maps.Clone`: https://github.com/docker/buildx/actions/runs/13858143332/job/38779661844#step:4:528

```
126.0 util/imagetools/loader.go:131:4: Replace m[k]=v loop with maps.Clone
```

But `modernize --fix` does a `maps.Copy`.